ui/components/Navigator.js

```diff
@@ -147,13 +147,11 @@ const styles = (theme) => ({
     overflowX : "hidden",
     transition : theme.transitions.create("width", { easing : theme.transitions.easing.sharp,
       duration : theme.transitions.duration.enteringScreen, }), },
-  fixedSidebarFooter : { display : "none",
-    [theme.breakpoints.up("sm")] : {
-      display : "flex",
-      flexDirection : "column",
-      "margin-top" : "auto",
-      "margin-bottom" : "0.5rem",
-    }, },
+  fixedSidebarFooter : { display : "flex",
+    flexDirection : "column",
+    "margin-top" : "auto",
```

Comment:
lets not mix up the convention, we are using camel case everywhere for declaring style properties, let's just use the same

Reply:
I have just removed some lines, didn't add any! So there is no change in convention because of my changes!

Reply:
Ok, no issues @yagarwal1307 ✌🏽  , can you change them to camel casing ?